skills/shadcn-vue/cli.md

```
@@ -0,0 +1,228 @@
+
+### `docs` — Get component documentation URLs
+
+```bash
+npx shadcn-vue@latest docs <components...> [options]
+```
+
+Outputs resolved URLs for component documentation, examples, and API references. Accepts one or more component names. Fetch the URLs to get the actual content.
+
+Example output for `npx shadcn-vue@latest docs input button`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)